src/assets/less/common.less

```diff
@@ -106,28 +106,64 @@ img {
 
 .modal .tab-content {
   height: 100%;
-  max-height: calc(100vh - 300px);
-  overflow-y: scroll;
+  max-height: 98%;
+  overflow-y: auto;
 }
 
 #settings-modal .modal-content {
-  height: 671px;
+  height: 90vh;
+  max-height: 90vh;
 }
 
 .modal .settings-group {
-  text-align: center;
-  max-width: 250px;
-  margin-left: auto;
-  margin-right: auto;
+    text-align: center;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    margin-left: auto;
+    margin-right: auto;
 }
-.modal .settings-group > div {
-  float: left;
-  clear: both;
+
+
+div.custom-control {
+  position: relative;
+    display: flex;
+    flex-wrap: wrap;
+    min-height: 1.40625rem;
+    cursor: pointer;
+    margin: 1%;
+    width: 23%;
+    height: fit-content;
+    padding-left: 1.5rem;
+}
+
+#components-checks .custom-control {
+    text-align: left;
+    width: 30%;
+    min-width: 23%;
 }
 
+
+.custom-radio > input, .custom-radio > label {
```

Comment:
Please use more specific names than `.custom-` prefix

Reply:
I generally don't prefer to use .custom prefix but since this CSS class was being already used, I decided to re-use it instead of creating a new separate class